deployment/ubuntu-package-armhf/docker-build.sh

```diff
@@ -18,3 +18,4 @@ dpkg-buildpackage -us -uc -aarmhf
 # Move the artifacts out
 mkdir -p ${ARTIFACT_DIR}/deb
 mv /jellyfin_* ${ARTIFACT_DIR}/deb/
+chown -Rc $(stat -c %u:%g ${ARTIFACT_DIR}) ${ARTIFACT_DIR}
```

Comment:
Do these `%u` and `%g` reflect the same user as `${current_user}`?

Reply:
Yes.
What this does, is taking the `UID` and `GID` of `${ARTIFACT_DIR}` (which is created outside of docker), and applying it recursively to the content.